Bottom: 411076dc3f01abdaea69ac5857900ce5b9d70a98
Top:    1a127c4b23b35fa597adc1323021353f2fb2346c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:30:36 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 5817469..f79a12f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -467,9 +467,10 @@ object3d_from_board_outline (void)
           if (ct->is_round)
             {
 
-              face3d_set_cylindrical (faces[i], COORD_TO_MM (ct->cx), COORD_TO_MM (ct->cy), 0., /* A point on the axis of the cylinder */
-                                                0., 0., 1.,                                     /* Direction of the cylindrical axis */
-                                                COORD_TO_MM (ct->radius));
+              face3d_set_cylindrical (faces[i],
+                                      COORD_TO_MM (ct->cx), COORD_TO_MM (ct->cy), 0., /* A point on the axis of the cylinder */
+                                      0., 0., 1.,                                     /* Direction of the cylindrical axis */
+                                      COORD_TO_MM (ct->radius));
               face3d_set_normal (faces[i], 1., 0., 0.);  /* A normal to the axis direction */
                                         /* XXX: ^^^ Could line this up with the direction to the vertex in the corresponding circle edge */
